test/fixtures/sourcemap/full.apib

```diff
@@ -0,0 +1,121 @@
+Format: 1A
+
+# <API name>
+<API description>
+
+# Group <resource group name>
+<resource group description>
+
+## <resource name> [/resource/{parameter}]
+<resource description>
+
++ Attributes (object)
+
+    + `<attribute property name>`: `<attribute property value>` (string) - <attribute property description>
+
++ Model
+
+    <resource model description>
+
+    + Headers
+
+            header1: <header1 value>
+
+    + Body
+
+            <resource model body>
+
+    + Schema
+
+            <resource model schema>
+
++ Parameters
+    + parameter = `<default value1>` (<type>, optional, `<example value1>`) ... <description>
+
+        + Values
+            + `<default value2>`
+            + `<example value2>`
+            + `<WTF value>`
+
++ Headers
+
+        header2: <header2 value>
+
+### <action name> [POST /users/{id}/edit]
+<action description>
+
++ Relation: relation-name
+
++ Parameters
+    + parameter = `<default value>` (<type>, optional, `<example value>`) ... <description>
+
+        + Values
+            + `<default value>`
+            + `<example value>`
+
++ Headers
+
+        header3: <header3 value>
+
++ Request <request name> (text/plain)
+
+    <request description>
+
+    + Headers
+
+            header4: <header4 value>
+
+    + Body
+
+            <request body>
+
+    + Schema
+
+            <request schema>
+
++ Response 200 (text/plain)
+
+    <response description>
+
+    + Headers
+
+            header5: <header5 value>
+
+    + Body
+
+            <response body>
+
+    + Schema
+
+            <response schema>
+
++ Response 201
+
+    [<resource name>][]
+
++ Response 201 (application/json)
+
+    + Attributes (<data structure name>)
```

Comment:
This PR is missing integration tests. Instead of just copying the blueprint file from `features/fixtures`, please add an integration test similar to how I added for ParseResult refract.

Please remove the above lines and just use a small sample in here.